org/allPRs.ts

```diff
@@ -1,6 +1,10 @@
 import { danger, warn, fail, GitHubCommit, markdown } from "danger"
-
 import yarn from "danger-plugin-yarn"
+import { ParseResult } from "./helpers/changelog/changelog-types"
+// @ts-ignore
+import { parsePRDescription } from "./helpers/changelog/parsePRDescription.js"
```

Comment:
why do we need ts-ignore here?

Reply:
typescript had issues importing javascript files. Maybe I can try to update the typescript config file to fix that as well